utils/utils.go

```diff
@@ -14,6 +17,13 @@ import (
 	sentry "github.com/getsentry/sentry-go"
 )
 
+type GitHubRelease struct {
```

Comment:
Do you need these changes in this PR any more..? I think this not needed right?

Reply:
Yep removed them from this PR